### DISCUSSION TITLE
Is it possible to remove or disable the watermark from the screenshot?

### DISCUSSION THREAD

**Discussion by chihebnabil** [hizzgdev/jsmind#593] [General]
Hello !

I don't see any information in the documentation about removing, disabling, or changing the watermark from the screenshot plugin

**Comment by hizzgdev** [comment]
Hi @chihebnabil
Since screenshot is an experimental feature, I didn't introduce the detail of it in the doc. Actually there is a way to resolve your problem.

```javascript
let options = {
    container: '',
    // ... options
    plugin: {
        screenshot: {
            filename: null,
            watermark: {
                left: 'text-left',
                right: 'text-right',
            },
            background: 'transparent'
        }
    }
}
let jm = new jsMind(options);
```

It's strongly recommended that you read [Experimental Features](https://github.com/hizzgdev/jsmind/blob/master/docs/en/experimental-features.md) to fully understand the risks before using this feature.
